_posts/2024-02-18-how-its-made.md

```diff
@@ -0,0 +1,185 @@
+---
+title: >-
+  How the sausage is made, or: How OpenTTD is developed
+author: michi_cc
+---
+
+Ever wondered how a new OpenTTD release is made?
+How we decide what features to include and what to reject or how some people to seem to know the "future" before you?
+Curious what it means that OpenTTD is Open Source?
+Or maybe you've even wondered what it takes to get your idea included in OpenTTD?
+
+In this post, you'll get a peek behind the quite transparent curtain and learn more on how OpenTTD is developed.
+
+<!-- more -->
+
+## What's source anyway?
+
+Let's start with the meaning of Open Source, as the source (or source code) of a computer program really is the source of it all.
+In a nutshell, the source code of a computer program is the human-readable form of any program.
```

Comment:
While it's the common way to phrase it, I wonder if the term "human-readable" can also be confusing. I imagine someone thinking, "I'm a human, and I certainly can't grok C++."
Mayhaps something like, "The source code is the form a program is written in by humans, and is meaningful when making modifications to it too."

Reply:
Tried a different phrasing.